timon/plugins/http_server/views.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins.http_server.views
+"""
+Summary: The http server plugin views and routes
+"""
+# #############################################################################
+import logging
+
+from quart_trio import QuartTrio
+
+from timon.probes.probe_if import mk_probe
+
+logger = logging.getLogger(__name__)
+
+app = QuartTrio(__name__)
+
+KNOWN_ROUTES = {
+    "/resources/": "returns a list of all resources and their availability",
+    "/heap/": "returns the lenght of the heap",
```

Comment:
why do you call it `heap` if it does not return the heap but only its size?

Reply:
url changed